<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mantid MSlice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <widget class="DataLoaderWidget" name="data_loading">
        <attribute name="title">
         <string>Data Loading</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="workspace_manager">
        <attribute name="title">
         <string>Workspace Manager</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="WorkspaceManagerWidget" name="wgtWorkspacemanager" native="true">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="controls" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="1">
                <widget class="QPushButton" name="btnRename">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Rename</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="6" column="1">
                <widget class="QPushButton" name="btnSaveToADS">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Save to MantidPlot</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QPushButton" name="btnDelete">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="btnAdd">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="btnSave">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="btnPlot">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Plot</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QPushButton" name="btnMerge">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Merge</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="btnOverplot">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Overplot</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="btnSubtract">
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Subtract</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="7" column="1">
                <widget class="QPushButton" name="btnHistory">
                 <property name="text">
                  <string>History</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="PowderWidget" name="wgtPowder">
               <attribute name="title">
                <string>Powder</string>
               </attribute>
              </widget>
              <widget class="SliceWidget" name="wgtSlice">
               <attribute name="title">
                <string>Slice</string>
               </attribute>
              </widget>
              <widget class="CutWidget" name="wgtCut">
               <attribute name="title">
                <string>Cut</string>
               </attribute>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuDefault_Energy_Units">
     <property name="title">
      <string>Default Energy Units</string>
     </property>
     <addaction name="actionmeV"/>
     <addaction name="actioncm"/>
    </widget>
    <addaction name="menuDefault_Energy_Units"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionmeV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>meV</string>
   </property>
  </action>
  <action name="actioncm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cm-1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WorkspaceManagerWidget</class>
   <extends>QWidget</extends>
   <header>mslice/widgets/workspacemanager/workspacemanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliceWidget</class>
   <extends>QWidget</extends>
   <header>mslice/widgets/slice/slice.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CutWidget</class>
   <extends>QWidget</extends>
   <header>mslice/widgets/cut/cut.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PowderWidget</class>
   <extends>QWidget</extends>
   <header>mslice/widgets/projection/powder/powder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DataLoaderWidget</class>
   <extends>QWidget</extends>
   <header>mslice/widgets/dataloader/dataloader.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
